src/main/kotlin/com/ampnet/blockchainapiservice/controller/ContractDeploymentRequestController.kt

```diff
@@ -76,6 +76,19 @@ class ContractDeploymentRequestController(
         )
     }
 
+    @GetMapping("/v1/deploy/by-project/{projectId}/by-alias/{alias}")
+    fun getContractDeploymentRequestByProjectIdAndAlias(
+        @PathVariable("projectId") projectId: UUID,
+        @PathVariable("alias") alias: String
+    ): ResponseEntity<ContractDeploymentRequestResponse> {
+        val contractDeploymentRequest = contractDeploymentRequestService
+            .getContractDeploymentRequestByProjectIdAndAlias(
+                projectId,
```

Comment:
Add parameter names to function call here: `projectId = projectId, alias = alias`